LightBulb/Services/SettingsService.cs

```diff
@@ -63,6 +63,9 @@ public partial class SettingsService() : SettingsBase(GetFilePath())
     [ObservableProperty]
     private double _configurationTransitionOffset;
 
+    [ObservableProperty]
+    private TimeSpan _configurationSmoothingDuration = TimeSpan.FromSeconds(15);
```

Comment:
Final thing: let's rename it to `_configurationSmoothingMaxDuration` since it's a cap and not all durations will be the same

Reply:
Ye, done.